Tests/CommandsTests/WorkspaceTests.swift

```diff
@@ -157,6 +157,8 @@ final class WorkspaceTests: XCTestCase {
             // Load the "current" manifests.
             let manifests = try workspace.loadDependencyManifests()
             XCTAssertEqual(manifests.root.package, graph.rootManifest.package)
```

Comment:
Heh -- I love how this change from a tuple to a struct and source didn't change. 👍